docs/Credits.md

```diff
@@ -42,7 +42,8 @@ Unless otherwise specified, all the following are from [the Noun Project](https:
 * [Bow](https://thenounproject.com/search/?q=bow&i=101736) By Arthur Shlain for Bowman
 * [Fishing Vessel](https://thenounproject.com/term/fishing-vessel/23815/) By Luis Prado for Work Boats
 * [Greek Trireme](https://thenounproject.com/search/?q=ancient%20boat&i=1626303) By Zachary McCune for Trireme
-* [Greek Trireme](https://thenounproject.com/search/?q=ancient%20boat&i=1626303) By Zachary McCune for Quinquereme
+* [Greek Trireme](https://thenounproject.com/search/?q=ancient%20boat&i=1626303) By Zachary McCune for Quinquereme // These credits are incorrect
+* [Norse Longship](https://thenounproject.com/zmccune/uploads/?i=1626302) by Zack McCune for Dromon
```

Comment:
Way too Viking.

Reply:
I looked at the wikipedia for general expectations and scanned pages of tnp icons - those with rows of oars are rare to begin with, and so I came up empty. Another gimp maybe.   (as long as it's not [this one](https://thenounproject.com/term/boat/1611891) )

Reply:
I have to say that gimping isn't my strong suit, so I've for now taken a TNP image that roughly resembles the icon of the original

Reply:
Why not - it's not a ship, but readily recognizable as different from other ships. And cute, too.